<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCC::KillSwitchAlertDialog</class>
 <widget class="QDialog" name="OCC::KillSwitchAlertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kill Switch Triggered</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="styleSheet">
      <string>QFrame { background-color: #FF4444; border-radius: 8px; padding: 10px; }</string>
     </property>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="styleSheet">
         <string>font-size: 32px;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="headerTextLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>KILL SWITCH ACTIVATED</string>
          </property>
          <property name="styleSheet">
           <string>QLabel { color: white; font-size: 18px; font-weight: bold; }</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="subtitleLabel">
          <property name="text">
           <string>Synchronization has been paused to protect your data</string>
          </property>
          <property name="styleSheet">
           <string>QLabel { color: white; }</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="reasonGroup">
     <property name="title">
      <string>Reason</string>
     </property>
     <layout class="QVBoxLayout" name="reasonLayout">
      <item>
       <widget class="QLabel" name="reasonLabel">
        <property name="text">
         <string>Threat detected</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="threatGroup">
     <property name="title">
      <string>Detected Threats</string>
     </property>
     <layout class="QVBoxLayout" name="threatLayout">
      <item>
       <widget class="QLabel" name="affectedFilesLabel">
        <property name="text">
         <string>0 threat(s) detected</string>
        </property>
        <property name="styleSheet">
         <string>color: gray;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="threatList">
        <property name="styleSheet">
         <string>QListWidget { font-family: monospace; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="text">
      <string>Only reset the Kill Switch if you are certain this was a false positive. If you are unsure, keep protection active and investigate the detected threats.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: #FF6600; font-style: italic; padding: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="keepProtectionButton">
       <property name="text">
        <string>Keep Protection Active</string>
       </property>
       <property name="styleSheet">
        <string>QPushButton { background-color: #4CAF50; color: white; font-weight: bold; padding: 10px 20px; border-radius: 5px; }
QPushButton:hover { background-color: #45a049; }</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset and Resume Sync</string>
       </property>
       <property name="styleSheet">
        <string>QPushButton { background-color: #f44336; color: white; padding: 10px 20px; border-radius: 5px; }
QPushButton:hover { background-color: #da190b; }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
